playbooks/domon-https.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Create objects in Icinga
+  hosts: localhost
+  gather_facts: false
+  vars:
+    object_name: domon-https
+  tasks:
+    # icinga_service_template
+    - name: "Icinga_service_template-{{ object_name }}"
+      ansible.builtin.include_role:
+        name: "telekom_mms.icinga_director.ansible_icinga"
+        tasks_from: "icinga_service_template"
+      vars:
+        icinga_service_templates:
+          - service_template_object:
+              - "{{ object_name }}"
+            imports:
+              - domon-non-agent-service
+            check_command: http
+            check_interval: 300
+            vars:
+              http_address: $check_address$
+              http_extendedperfdata: false
+              http_invertregex: false
+              http_sni: true
+              http_ssl: true
+              http_timeout: 30
+          - service_template_object:
+              - "{{ object_name }}-certificate"
+            imports:
+              - "{{ object_name }}"
+            check_period: only_at_1000-1005
```

Comment:
This is not a default check_period. Either we leave this blank or set a default with a variable that a user can change.

Reply:
I left this blank, as suggested